backend/app/pydantic_models/notes.py

```
@@ -1 +1,11 @@
 # do something here bruh
+from pydantic import BaseModel
```

Pylint (3.3.7)

[error] Unable to import 'pydantic'

(E0401)